contributing-a-devnote.md

```diff
@@ -11,21 +11,275 @@ DevNotes are articles from our community that are published here on the Nucleus
 
 # Let's get started
 
 The remainder of this document is going to take you through getting started and submitting your first DevNote. There are links out to resources online that will help you complete tasks or provide useful context information.
 
-1. Installing the Curvenote Command line. Using conda. Using pip. Using npm. (have links)
+## Installing the Curvenote Command line.
+
+When writing your DevNote you can choose to work locally in either the `mystmd` CLI or the `curvenote` CLI tool. Both of these tools will allow you to run a local read and review your markdown changes in a web browser.
+
+However, the Curvenote CLI is required in order to submit a draft and to fully preview your DevNote on the Nucleus DevNote website. However, if you're interested in installing `mystmd` follow the instructions at https://mystmd.org/guide/installing.
+
+To install the If you have no CLI, you will first need to install Node.js on your system. You may already have this installed, so let's first check.
+
+### Confirm node installation
+
+In a terminal or command line prompt, run the following commands.
+
+```sh
+node -v
+npm -v
+```
+
+Here you have Node version 18 or above and NPM version 9 or above available on your system. Here you have Node version 18 or above and NPM version 9 or above available on your system. If you don't have Node.js installed you can install it by following the instructions here: https://curvenote.com/docs/publish/installing-prerequisites.
+
+### Install the curvenote CLI
+
+Once you've ensured that you have node.js available, you can install the Curvenote CLI using the Node Package Manager, npm. To do that, run the following command ensuring that you include the `-g` flag as shown.
+
+```sh
+npm install -g curvenote@latest
+```
+
+Confirm the installation by running the following code:
+
+```sh
+curvenote -v
+```
+
+## Getting your API token
+
+In order to fully use the Curvenote CLI and to preview and submit your DevNote you will need to create an API Token and register that with the Curvenote CLI on your machine.
+
+To generate your token, follow the instructions here: https://curvenote.com/docs/publish/authentication
+
+You can confirm and check the validity of your API token at any time by running the following command on your machine:
+
+```sh
+curvenote token check
+```
+
+Which, for a valid token, will display something like this
+
+```sh
+Active token:
+"Steve Purves" <amolina@bnextbio.com> at https://api.curvenote.com/login
+Expiry: no expiry
+Token status: VERIFIED
+Login as @stevejpurves <amolima@bnextbio.com> verified by https://sites.curvenote.com/v1/my/user
+```
+
+## Install text editor
+
+We recommend installing [VSCode](https://code.visualstudio.com/download).
+
+## Create a Github account and set up SSH
+
+## Clone the DevNote template repository
+
+STEVE: I suggest we orient this back around the devnotes central repo, copying your template into place there?
```

Comment:
@antonrmolina Awaiting on your answers on this point (see slack) before continuing to finalize the submission parts